src/main.css

```diff
@@ -8,3 +8,12 @@
 .jumbotron {
```

Comment:
Very good job John. Please make sure you discuss with the team if they are comfortable using tailwind.

Reply:
Sure I will. We shall plan and schedule when we shall be meeting for one to discuss with the team what they worked on and how and what tools they used. Thanks